gosnowflake.mak

```diff
@@ -1,32 +1,35 @@
 ## Setup
+SHELL := /bin/bash
+SRC = $(shell find . -type f -name '*.go' -not -path "./vendor/*")
+
 setup:
 	go get golang.org/x/crypto/ocsp
```

Comment:
can you remove this line as well? This has been redundant, and it is good chance to remove it.

Reply:
Which line? `go get golang.org/x/crypto/ocsp`?

Reply:
Yes `go get .../ocsp`. The package should be updated by `dep`